app/controllers/courses_controller.rb

```diff
@@ -41,6 +41,13 @@ def duplicate
     redirect_to @course.copy, flash: { success: t('courses.copied_succesfully') }
   end
 
+  def destroy
+    @course = Course.find(params[:id])
+    @course.destroy
+
+    redirect_to action: 'index'
```

Comment:
To make it a bit more responsive to the user, you can add a flash:

```suggestion
    redirect_to action: 'index', flash: { success: t('courses.deleted_successfully', course.name) }
```
And add to Lin 32 in config/locales/en.yml something like:

`deleted_successfully: The course %{name} has been deleted successfully`

See https://guides.rubyonrails.org/action_controller_overview.html#the-flash

Reply:
It did not work with `flash: { success: t('courses.deleted_successfully', course.name) }`, so I used `flash[:notice] = "The course #{@course.name} has been deleted successfully" ` instead and did not modify `en.yml`. It appears to be working fine.